integration/vite-dev-express-test.ts

```diff
@@ -141,16 +143,20 @@ test("Vite custom server HMR & HDR", async ({ page }) => {
 
   // setup: browser state
   let hmrStatus = page.locator("#index [data-hmr]");
+  await expect(page).toHaveTitle("HMR updated title: 0");
   await expect(hmrStatus).toHaveText("HMR updated: 0");
   let input = page.locator("#index input");
   await expect(input).toBeVisible();
   await input.type("stateful");
 
   // route: HMR
   await edit("app/routes/_index.tsx", (contents) =>
```

Comment:
Just noticed you added an `edit` util. I was thinking of writing this too. This is super nice.